hashintel/hash
# Restyle Countdown block

## 🌟 What is the purpose of this PR?



This PR updates the styling of countdown block
- [Figma Designs](https://www.figma.com/file/T10d2P2iNb1LdRC5a9J7vA/BP-Block-Week-Designs?node-id=49%3A9164) (internal link)

## 🔗 Related links





- [Implement Countdown block designs](https://app.asana.com/0/1202176479837585/1202255059566946) _(internal)_


## 🔍 What does this change?




- Updates the countdown block UI to match designs
- Breaks the existing app into smaller components 
- Removes range selection feature. [internal link](https://hashintel.slack.com/archives/C02LG39FJAU/p1653317467951639) for more context
- Adds a `displayTime` prop which is used to determine if time (hours, mins, seconds) should also be displayed in the countdown

## 📜 Does this require a change to the docs?



-No

## ⚠️ Known issues


- The time input makes use of the native time picker and it's not possible to override the picker styles to match our designs on chrome. We'd need to create a custom time picker instead.

<img width="306" alt="Screenshot 2022-05-25 at 15 38 57" src="https://user-images.githubusercontent.com/16928688/170288813-7f4181ce-cf5d-482d-9382-0b29fc1cca24.png">

## 🐾 Next steps


- Create a custom time picker

## ❓ How to test this?



1.  Run the block locally or load into the workspace app
1.  Confirm it matches attached design and functionality is the same

## 📹 Demo



https://user-images.githubusercontent.com/16928688/170277498-f942d02c-e7bd-4139-bacb-f134bb256ac1.mov
